content/_index.md

```diff
@@ -65,14 +65,14 @@ gantt
 - dbt CLI installed / dbt Cloud account available
   - New dbt project created and configured
 - Python 3 Installed
-- Snowflake account and a user with access to create schemas and tables
+- Snowflake / Databricks account and a user with access to create schemas and tables
```

Comment:
Could we change this to Snowflake or Databricks account? It might look like they need both for this

Reply:
I also changed line 65 as it is the same situation there. For me the / means optional or choice but for sure let's change them if it causes ambiguity.